src/lib/models/KQStream.ts

```diff
@@ -0,0 +1,135 @@
+interface Position {
+  x: number;
+  y: number;
+}
+
+export enum Character {
+  GoldQueen = 1,
+  BlueQueen = 2,
+  GoldStripes = 3,
+  BlueStripes = 4,
+  GoldAbs = 5,
+  BlueAbs = 6,
+  GoldSkulls = 7,
+  BlueSkulls = 8,
+  GoldChecks = 9,
+  BlueChecks = 10,
+}
+
+export enum Team {
+  Gold = 'GOLD',
+  Blue = 'BLUE',
+}
+
+export enum Maiden {
+  Warrior = 'WARRIOR',
+  Speed = 'SPEED',
+}
+
+export enum GameMap {
+  Day = 'DAY',
+  Night = 'NIGHT',
+  Dusk = 'DUSK',
+}
+
+// Read keys as LeftRight
+export enum CabOrientation {
+  BlueGold = 'BLUE_GOLD',
+  GoldBlue = 'GOLD_BLUE',
```

Comment:
I think these constants would be more readable if they were `BlueOnLeft` and `GoldOnLeft`, or something equivalent.